src/sb-util.ts

```diff
@@ -19,16 +21,23 @@ import {
 import { map, filter, makeIterable, first } from './generators';
 import { BlockOpcodeToShape } from './block-shapes';
 
-enum ScratchProjectKeys {
-    TARGETS = 'targets',
-}
-
 enum SpriteAttributes {
     BLOCKS = 'blocks',
     BROADCASTS = 'broadcasts',
     LISTS = 'lists',
 }
 
+/**
+ * A default (broken) asset fetcher
+ */
+const NoAssetFetcher: AssetFetcher = {
```

Comment:
I think this should be at least an anonymous class for debugging.

```js
const NoAssetFetcher = new class {
```

Reply:
I'd prefer NoAssetFetch to instantiated on demand, but I'm not sure it weighs enough in either direction to tell if that'd be better.

Reply:
rather than fix this contention now, i just removed this stub i never used